kubernetes/loculus/templates/_inputFieldsFromValues.tpl

```diff
@@ -2,7 +2,7 @@
 {{- $data := . }}
 {{- $metadata := $data.metadata }}
 {{- $extraFields := $data.extraInputFields }}
-{{- $TO_KEEP := list "name" "displayName" "definition" "guidance" "example" "required" "noEdit"}}
+{{- $TO_KEEP := list "name" "displayName" "definition" "guidance" "example" "required" "noEdit" "desired"}}
```

Comment:
It would be great if you could add desired also to https://loculus.org/reference/helm-chart-config/#metadata-type, and could you maybe add some description for the other, related fields like guidance and definition?